src/main/java/training/subsystems/Roller/RollerSubsystem.java

```diff
@@ -0,0 +1,67 @@
+package training.subsystems.Roller;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import utils.GBSubsystem;
+
+import static training.subsystems.Roller.Constants.*;
+
+public class RollerSubsystem extends GBSubsystem {
+    private static RollerSubsystem instance;
+    private final CANSparkMax motor = new CANSparkMax(ROLLER_ID, CANSparkLowLevel.MotorType.kBrushless);
+    private final SparkPIDController motorPID = motor.getPIDController();
+    private double velocity;
```

Comment:
```suggestion
    private double targetVelocity;
```
also can be with rot2d